.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.vscode/
```

Comment:
I do not like to change the files from the course after creating the video so could you remove the changes from this file.